From 7130850d74863d1e9a0ae3e85462f1466bdf6d88 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Thu, 14 Nov 2019 10:13:53 +0800
Subject: [PATCH] dont setup compiler flags -m32/-m64

We don't want these to be setup by boost as we pass our own flags.

Upstream-Status: Inappropriate [OE-specific]

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 tools/build/src/tools/gcc.jam | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/tools/build/src/tools/gcc.jam b/tools/build/src/tools/gcc.jam
index ed7e39baf..90513f84a 100644
--- a/tools/build/src/tools/gcc.jam
+++ b/tools/build/src/tools/gcc.jam
@@ -377,22 +377,6 @@ local rule compile-link-flags ( * )
     compile-link-flags <link>shared/<target-os>$(non-windows) : -fPIC ;
 }
 
-{
-    # Handle address-model
-    compile-link-flags <target-os>aix/<address-model>32 : -maix32 ;
-    compile-link-flags <target-os>aix/<address-model>64 : -maix64 ;
-
-    compile-link-flags <target-os>hpux/<address-model>32 : -milp32 ;
-    compile-link-flags <target-os>hpux/<address-model>64 : -mlp64 ;
-
-    local generic-os = [ set.difference $(all-os) : aix hpux ] ;
-    compile-link-flags <target-os>$(generic-os)/<architecture>s390x/<address-model>32 : -m31 ;
-    compile-link-flags <target-os>$(generic-os)/<architecture>s390x/<address-model>64 : -m64 ;
-    local arch = power sparc x86 ;
-    compile-link-flags <target-os>$(generic-os)/<architecture>$(arch)/<address-model>32 : -m32 ;
-    compile-link-flags <target-os>$(generic-os)/<architecture>$(arch)/<address-model>64 : -m64 ;
-}
-
 {
     # Handle threading
     local rule threading-flags ( * )
